mxcube3/ui/components/SampleView/ContextMenu.js

```diff
@@ -99,11 +100,27 @@ export default class ContextMenu extends React.Component {
   showModal(modalName, wf = {}, _shape = null) {
     const { sampleID, defaultParameters, shape, sampleData } = this.props;
     const sid = _shape ? _shape.id : shape.id;
+    const cellCount = shape.gridData ? defaultParameters[modalName.toLowerCase()].cell_counting
+      : 'None';
+
+    let aux;
```

Comment:
Is more that num_images will be aux... the num_images is determined by the num_{cols, rows} and the cell-counting method and then fixed after so the user cannot change it.